src/ome_zarr_models/v04/coordinate_transformations.py

```diff
@@ -26,7 +27,18 @@ class VectorScale(Base):
     """
 
     type: Literal["scale"]
-    scale: list[float] = Field(..., min_length=2)
+    scale: list[float]
+
+    @classmethod
+    def build(cls, data: Iterable[float]) -> Self:
```

Comment:
Could we instead give `type` a defualt (`type: Literal["scale"] = "scale"`), and then if you want to construct one of these you can just do `VectorScale(scale=[1, 2, 3])`? I think that's slightly nicer API and avoids us having to write a `build` method.

Reply:
the argument against giving the type a default value was that this would allow a JSON object without a `type` field to pass validation, even though that JSON object is technically an invalid axis